molecule/check_new_topology/converge.yml

```diff
@@ -0,0 +1,194 @@
+---
+
+- name: 'Change URI of core-1 instance'
+  hosts: core-1-replica
+  become: true
+  become_user: root
+  gather_facts: false
+  tasks:
+    - name: 'Set new advertise_uri'
+      set_fact:
+        config:
+          advertise_uri: 'vm1:3305'
+          http_port: 8105
+
+    - vars:
+        cartridge_force_advertise_uris_change: true
+        cartridge_scenario:
+          - configure_instance
+          - restart_instance
+          - wait_instance_started
+          - connect_to_membership
+          - edit_topology
+      import_role:
+        name: ansible-cartridge
+
+- name: 'Set default scenario'
```

Comment:
What about adding a custom scenario instead of this?

Reply:
Yes